addons/web/static/src/scss/attachment_preview.scss

```diff
@@ -0,0 +1,181 @@
+
+    .o_attachment_progress_bar {
+        display: none;
+    }
+
+    .o_attachment_uploaded, .o_attachment_delete {
+        @include o-position-absolute(0, 0, 0, $left: auto);
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: 45px;
+    }
+
+    .o_attachment_delete {
+        background: desaturate(theme-color('primary'), 50%);
+        color: white;
+        cursor: pointer;
+        font-size: 20px;
+        transform: translateX(100%);
+        transition: all 0.3s ease 0s;
+
+        &:hover {
+            background: theme-color('primary');
+        }
+    }
+
+    &.o_attachment_uploading {
+        .o_attachment_progress_bar {
+            display: inline-block;
+            margin: 0 0 0 8px;
+            border-radius: 2px;
+            vertical-align: bottom;
+
+            > div {
+                font-size: 11px;
+                padding: 0 7px;
+            }
+        }
+
+        .o_attachment_delete, .o_attachment_uploaded {
+            display: none;
+        }
+    }
+
+    &:hover .o_attachment_delete {
+        transition: all 0.1s ease 0s;
+        transform: translateX(0);
+    }
+}
+
+.o_attachments_previews {
+    > .o_attachment {
+        width: 50%;
+
+        @include media-breakpoint-up(md) {
+            width: percentage(1/3);
+        }
+        @include media-breakpoint-up(lg) {
+            width: percentage(1/4);
+        }
+        @include media-breakpoint-up(xl) {
+            width: percentage(1/6);
+        }
+
+        .o_attachment_wrap {
+            padding: 0;
+            border-radius: 2px;
+            border: 1px solid gray('400');
+        }
+
+        .o_image_box {
+            overflow: hidden;
+            padding-top: 75%;
+
+            .o_attachment_image {
+                @include o-position-absolute(0, 0, 0, 0);
+                background-size: cover;
+                background-position: center;
+                transform: scale(1.05);
+                transition: all 0.3s ease 0s;
+            }
+
+            &:hover .o_attachment_image {
+                transform: scale(1);
+            }
+        }
+
+        .o_image_overlay {
+            @include o-position-absolute($top: 0, $left: 0, $right:0, $bottom:0);
+            @include o-hover-opacity($default-opacity: 0, $hover-opacity: 1);
+            background-image: linear-gradient(180deg, rgba(black, 0.2), rgba(black, 0.9));
+            transition: all 0.3s ease 0s;
+            padding: 10px;
+
+            .o_attachment_title {
+                @include o-position-absolute($bottom: 7px);
+                width: 80%;
+                @include o-text-overflow(inline-block);
+                font-size: $font-size-sm;
+            }
+
+            .o_attachment_delete_cross {
+                color: white;
+            }
+
+            .o_attachment_download {
+                @include o-position-absolute($right: 7px, $bottom:7px);
+            }
+        }
+    }
+}
```

Comment:
missing blank line

Reply:
Done. added a missing blank line at end of the file.